15LessonFifteen/Checkpoint3/css/style.css

```diff
@@ -0,0 +1,160 @@
+main>section:first-child {
+  background: gray url('https://gualerdz57.github.io/intro-workbook/15LessonFifteen/Checkpoint3/img/wine-bg.jpg') no-repeat center center;
+  background-blend-mode: multiply;
+  background-size:cover;
+}
+.border-around{
+
+  border-left:1px solid white;
+}
+.flex-container{
+  display:flex;
+}
+
```

Comment:
probably want something like `img{max-width:100%;}` . Right now the width of http://foundry.mediumra.re/img/wine1.png is not set and happens to be larger than 320; ergo you get horizontal scrolling @ the smallest screen size.